server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRToAuditList.java

```diff
@@ -157,32 +156,8 @@ public String endpointBody(final Request the_request, final Response the_respons
 
     try {
       // get the request parameters
-      final String start_param = the_request.queryParams(START);
```

Comment:
Do all of these optional parameters only apply if we're selecting from the CVRs?